bundles/org.openhab.io.metrics/README.md

```diff
@@ -0,0 +1,109 @@
+# Metrics service
+
+The metrics service provides 
+
+* an additional REST endpoint to retrieve openHAB core metrics from. This can be used as scrape target for pull-based monitoring systems like [Prometheus](https://prometheus.io/).
+* optionally configurable services to export openHAB core metrics to push-based monitoring systems like [InfluxDB](https://www.influxdata.com/). 
+
+## Precondition
+
+The openHAB core metrics must be available - a recent enough openHAB version must be used.
+https://github.com/openhab/openhab-core/pull/2133
+
+## Provided metrics
+
+Currently the following metrics are provided: 
+
+- openHAB events counts (per topic)
+- openHAB bundle states
+- openHAB thing states
+- openHAB rule runs (per rule)
+- openHAB threadpool stats (per scheduler)
+- JVM stats including metrics of
+    - class loader
+    - memory
+    - GarbageCollector
+    - OS (system load, CPU)
+    - thread metrics
+
+## Configuration
+
+The configuration for the metrics service is available in the openHAB UI under Settings | Other Services | Metrics service.
+Support for pull-based monitoring systems (e. g. Prometheus) is always enabled, since it doesn't imply any significant overhead when not used.
+Support for push-based monitoring systems (e. g. InfluxDB) have to be enabled seperately. 
+
+The following configuration parameters can be set:
+<table>
+  <thead>
+    <tr><th>Config param</th><th>Description</th><th>Default value</th></tr>
+  </thead>
+  <tbody>
+    <tr><td>influxMetricsEnabled</td><td>Enable the Influx (www.influxdata.com) metrics. Further configuration of the InfluxDB instance necessary.</td><td>false</td></tr>
+  </tbody>
+</table>
```

Comment:
You could use markdown syntax for the tables.

Reply:
afaik it doesn't support col span which I think will greatly improve clarity here.

Reply:
That's right, markdown doesn't support colspan. Are you using colspan here?

Reply:
:rofl: good point. I think I had in the original version, that's why I introduced the HTML table in the first place, but it changed a lot since then. I'll re-markdown it.